config/security-bindings.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (C) 2011-2022 Red Hat, Inc. (https://github.com/Commonjava/indy-tracking-service)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+constraints:
+  - role: admin
+    urlPattern: "/api/admin/.*"
+    methods:
+      - POST
+      - PUT
+      - DELETE
+  - role: user
+    urlPattern: "/api/.*"
+    methods:
+      - POST
+      - PUT
+      - DELETE
+  - role: power-user
+    urlPattern: "/api/admin/stores/.*"
+    methods:
+      - POST
+      - PUT
+      - DELETE
```

Comment:
This security-bindings.yaml is used for authorization with keycloak authentication on API level. Maybe you should change the settings based on tracking service api. But this can be changed later. I'll send you the doc about our microservice authentication and authorization settings later.

Reply:
updated according to the endpoint identified in the implementation design